core/sbt-cloudflow/src/main/scala/cloudflow/sbt/BlueprintVerificationPlugin.scala

```diff
@@ -48,9 +48,20 @@ object BlueprintVerificationPlugin extends AutoPlugin {
             feedbackResults(verificationResult.value, log)
           }
         }.value,
+    allCloudflowStreamletDescriptors := (Def
+          .taskDyn {
+            val filter = ScopeFilter(inProjects(thisProject.value.uses: _*))
+            Def.task {
+              val allValues = cloudflowStreamletDescriptors.all(filter).value
+              allValues
+            }
+          })
+          .value
+          .flatten
+          .toMap,
```

Comment:
This is the real trick 👍 .. this `ScopeFilter` thingy ..

Reply:
It's still not exactly right for all cases, for instance where you have one project, which is both blueprint and streamlets lib.. (spark-sensors).. it would be better to just be able to find all of them by plugin..